### PR TITLE
Update src/library/TextInput

### PR DESCRIPTION
**Pull request by tx-steven**
Update FormField related styling

### CODE REVIEW

**Review by tx-michael — APPROVED** [review]

File: src/library/FauxControl/FauxControl.js [hunk 1]
```diff
@@ -24,12 +24,12 @@ const getIcons = ({
...
   iconStart,
   iconEnd,
-  readOnly,
+  // readOnly,
   size,
   variant,
   variantIcons
 }) => {
-  if (disabled || readOnly) {
+  if (disabled) { // Remove readOnly from condition to allow icons when input is in a readOnly state
```
Comment (tx-nestor) [inline src/library/FauxControl/FauxControl.js]: Remove the comment. I think the diff and the commented out prop above is sufficient to understand what's going on here.

File: src/library/FauxControl/styled.js [hunk 2]
```diff
@@ -137,7 +137,11 @@ export const Underlay = styled('div', {
     position: 'absolute',
     right: 0,
     top: 0,
-    zIndex: -1
+    zIndex: -1,
+
+    /* TargetX Custom Styles */
+    borderTopWidth: 0,
```
Comment (tx-nestor) [inline src/library/FauxControl/styled.js]: Let's remove these for now. I'm not comfortable or convinced that Ryan wants that box shadow.

File: src/library/Form/styled.js [hunk 3]
```diff
@@ -88,6 +88,9 @@ export const FormFieldTextWrapper = styled('div')(
     const theme = formFieldTheme(baseTheme);
 
     return {
+      /* TargetX Custom Styles */
```
Comment (tx-nestor) [inline src/library/Form/styled.js]: move to the bottom of the list to be consistent

**Review by tx-nestor — APPROVED** [review]
@tx-steven   Make requested changes, then go ahead merge

**Pull request merged**